spec/inputs/jdbc_spec.rb

```diff
@@ -23,10 +26,9 @@
     settings = {"statement" => "SELECT :num_param as num_param FROM SYSIBM.SYSDUMMY1", "parameters" => {"num_param" => 10} }
     plugin = LogStash::Inputs::Jdbc.new(mixin_settings.merge(settings))
     plugin.register
-    q = Queue.new
     plugin.run(q)
-    insist { q.size } == 1
-    insist { q.pop['num_param'] } == settings['parameters']['num_param']
+    expect(q.size).to eq(1)
```

Comment:
Why are you checking for the queue size, if you aim to check for the parameters? Don't understand why it's need.